pkg/flare/envvars.go

```diff
@@ -40,11 +40,25 @@ var envvarNameWhitelist = []string{
 	// Trace agent
 	"DD_APM_ENABLED",
 	"DD_APM_NON_LOCAL_TRAFFIC",
-	"DD_RECEIVER_PORT",
-	"DD_IGNORE_RESOURCE",
+	"DD_RECEIVER_PORT",   // deprecated
```

Comment:
There are some missing environment variables. Please add all of them from https://github.com/DataDog/datadog-agent/blob/3d874636a0dfb10539720bdf51be937fe6e335f9/pkg/trace/config/env.go. Everything you find in that file starting with `DD_*`.